tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -0,0 +1,216 @@
+# This file contains the data used to seed the test environment for the simulator tests
+
+data_set_id = 1521375514069
```

Comment:
There was a comment about having this in a fixture. The reason we have it here as a common variable is that we need it as part of the resources being seeded in this file (simulator integration) and models and routines in the other pr.